core/Node.js

```diff
@@ -1053,15 +1053,25 @@ Node.prototype.setOpacity = function setOpacity (val) {
  *                        y direction ("height").
  * @param {SizeMode} z    The size mode being used for determining the size in
  *                        z direction ("depth").
+ * @param {Function} customMode    The function used to calculate the custom
+ *                                 size.
+ * @param {Array} customParams     An array of parameters to pass into the 
+ *                                 custom calculation.  Updates to the original
+ *                                 array will be reflected in the next 
+ *                                 calculation.
  *
  * @return {Node} this
  */
-Node.prototype.setSizeMode = function setSizeMode (x, y, z) {
+Node.prototype.setSizeMode = function setSizeMode (x, y, z, customMode, customParams) {
```

Comment:
I'm not real satisfied with this... 5 input params is a lot so I am definitely open to other suggestions